src/orion/analysis/base.py

```diff
@@ -54,11 +57,17 @@ def average(trials, group_by="order", key="best", return_var=False):
         return trials
 
     group = trials.groupby(group_by)
-    mean = group[key].mean().reset_index().rename(columns={key: f"{key}_mean"})
-    if return_var:
-        mean[f"{key}_var"] = group[key].var().reset_index()[key]
+    means = list()
+    keys = [v.strip() for v in key.split(",")]
```

Comment:
This new behavior is not documented above.

Reply:
fix in https://github.com/Epistimio/orion/pull/685/commits/1cd9716b0b553da7f806a46a12c7dbb0171357b7